From 2041bc2cb5eb0fcad97d28d795f69b8419baf563 Mon Sep 17 00:00:00 2001
Message-Id: <2041bc2cb5eb0fcad97d28d795f69b8419baf563@dist-git>
From: Giuseppe Scrivano <gscrivan@redhat.com>
Date: Fri, 28 Mar 2014 22:54:04 +0100
Subject: [PATCH] doc: fix XML for the RNG device example

https://bugzilla.redhat.com/show_bug.cgi?id=786408

Add a missing '/' to close the "source" element.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>
(cherry picked from commit ab42c226278a1c1c5236c64472aef583aec39412)
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 docs/formatdomain.html.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/formatdomain.html.in b/docs/formatdomain.html.in
index e2e5b89..1856de7 100644
--- a/docs/formatdomain.html.in
+++ b/docs/formatdomain.html.in
@@ -4196,8 +4196,8 @@ qemu-kvm -net nic,model=? /dev/null
       &lt;backend model='random'&gt;/dev/random&lt;/backend&gt;
       &lt;!-- OR --&gt;
       &lt;backend model='egd' type='udp'&gt;
-        &lt;source mode='bind' service='1234'&gt;
-        &lt;source mode='connect' host='1.2.3.4' service='1234'&gt;
+        &lt;source mode='bind' service='1234'/&gt;
+        &lt;source mode='connect' host='1.2.3.4' service='1234'/&gt;
       &lt;/backend&gt;
     &lt;/rng&gt;
   &lt;/devices&gt;
-- 
1.9.1

